app/Http/Controllers/Admin/ProductController.php

```diff
@@ -275,11 +275,23 @@ public function dataTable()
                 return $currencyHelper->formatForDisplay($product->price);
             })
             ->editColumn('minimum_credits', function (Product $product, UserSettings $user_settings, CurrencyHelper $currencyHelper) {
-                return $product->minimum_credits ? $currencyHelper->formatForDisplay($product->minimum_credits) : $currencyHelper->formatForDisplay($user_settings->min_credits_to_make_server);
+                return $product->minimum_credits ? $currencyHelper->formatForDisplay($product->minimum_credits) : $user_settings->min_credits_to_make_server;
             })
             ->editColumn('serverlimit', function (Product $product) {
                 return $product->serverlimit == 0 ? "∞" : $product->serverlimit;
             })
+            ->editColumn('memory', function (Product $product) {
+                return $product->memory == 0 ? "∞" : $product->memory;
+            })
+            ->editColumn('cpu', function (Product $product) {
+                return $product->cpu == 0 ? "∞" : $product->cpu;
+            })
+            ->editColumn('swap', function (Product $product) {
+                return $product->swap == 0 ? "∞" : $product->swap;
+            })
```

Comment:
Please note that a swap value of 0 in Pterodactyl means that it will be disabled, not unlimited.
You should take into account that:
0 - Disabled.
-1 - Unlimited.
<img width="663" height="97" alt="image" src="https://github.com/user-attachments/assets/e1282450-ae9b-4c0c-b519-d7e2b565a47a" />